tests/chalicelib/resource_transfer.py

```diff
@@ -0,0 +1,47 @@
+from chalice import BadRequestError, Response
+from cuenca_validations.types import (
+    StrictTransferRequest,
+    TransactionStatus,
+    TransferQuery,
+)
+from mongoengine import NotUniqueError
+
+from tests.chalicelib._generic_query import generic_query
+from tests.chalicelib.model_transfer import Transfer as TransferModel
+
+from .base import app
+
+
+@app.resource('/mytest')
+class Transfer:
```

Comment:
similar al comentario anterior, no necesitas un resource complejo como este. No ayuda a leer lo esencial del diseño de este paquete.

también elige un nombre de recurso más genérico

Reply:
changed to more clear